presto-docs/src/main/sphinx/release/release-301.rst

```diff
@@ -0,0 +1,49 @@
+===========
+Release 301
+===========
+
+General Changes
+---------------
+
+* Fix reporting of aggregate input data size stats. (:issue:`100`)
+* Add support for role management (see :doc:`/sql/create-role`).  Note, using :doc:`/sql/set-role` requires an up-to-date client library. (:issue:`90`)
+* Add ``INVOKER`` security mode for :doc:`/sql/create-view`. (:issue:`30`)
+* Add ``ANALYZE`` SQL statement for collecting table statistics. (:issue:`99`)
+* Add :func:`log` function with arbitrary base. (:issue:`36`)
+* Remove the ``deprecated.legacy-log-function`` configuration option. The legacy behavior for the :func:`log` function is no longer available. (:issue:`36`)
+* Remove the ``deprecated.legacy-array-agg`` configuration option. The legacy behavior for :func:`array_agg` is no longer available. (:issue:`77`)
+* Disallow uninstalling Presto RPM while server is still running. (:issue:`67`)
+* Improve performance of `coalesce` expressions. (:issue:`35`)
+* Improve error message for unsupported :func:`reduce_agg` state type (:issue:`55`).
+* Improve performance of queries involving ``SYSTEM`` table sampling and computations over the columns of the sampled table. (:issue:`29`)
+
+Security
+--------
+
+* Support LDAP with anonymous bind disabled. (:issue:`97`)
+
+Hive Connector
+--------------
+* Add procedure for dumping Metastore recording to a file. (:issue:`54`)
+* Add Metastore recorder support to Glue. (:issue:`61`)
+* Add ``hive.temporary-staging-directory-enabled`` configuration property and ``temporary_staging_directory_enabled`` session property to control
+  whether a temporary staging directory should be used for write operations. (:issue:`70`)
+* Add ``hive.temporary-staging-directory-path`` configuration property and ``temporary_staging_directory_path`` session property to control the location
+  of temporary staging directory that is used for write operations. Use ${USER} placeholder to use different location for each user. (:issue:`70`)
+
+Base-JDBC Connector API
```

Comment:
We should decide if we want to start document this. It sends the message that this is a supported library for public consumption. If we do, let's name it as below, and move it to be after SPI changes.

```
Base-JDBC Connector Library Changes